server/api.go

```diff
@@ -833,6 +837,25 @@ func (p *publishAsyncSession) dispatchAcks() error {
 			p.inflight = 0
 		}
 		p.mu.Unlock()
+
+		// AckError indicates that an error is sent back via AckInbox
+		switch ack.AckError {
+		case client.Ack_OK:
+			break
+		case client.Ack_INCORRECT_OFFSET:
+			p.sendPublishAsyncError(ack.CorrelationId, &client.PublishAsyncError{
+				Code:    client.PublishAsyncError_INTERNAL,
```

Comment:
This should probably be a different error code as `INTERNAL` indicates an internal server error but this is really an OCC rejection. Perhaps we need to introduce a new error code?

Reply:
I am adding  a new PR for the API

Reply:
https://github.com/liftbridge-io/liftbridge-api/pull/48

PR opened for this matter